### DISCUSSION TITLE
Please How can I emulate timezones and battery percentage?

### DISCUSSION THREAD

**Discussion by alimayorru** [kaliiiiiiiiii/Selenium-Profiles#96] [Q&A] [answered]
1. Please How can I emulate timezones and battery percentage?

2. https://www.whatismybrowser.com/detect/what-operating-system-do-i-have still detects my real operating system even after changing platform on the profile

**Comment by ttlns** [comment]
@alimayorru 
1. this isn't implemented yet
2. Please provide the script you used, your operating system and  the system it detected

**Reply by alimayorru** [reply]
My OS is Windows 10, and that was detected



```python
import time

from selenium_profiles.webdriver import Chrome
from selenium_profiles.profiles import profiles
from selenium_driverless.webdriver import ChromeOptions
from selenium_driverless.types.by import By


profile = profiles.Windows()  # or .Android


profile["cdp"]["useragent"]["platform"] = "Linux"
profile["cdp"]["useragent"]["userAgent"] = "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/99.0.4844.84 Safari/537.36"

options = ChromeOptions()

driver = Chrome(profile, options=options, driverless_options=True)

# get url
driver.get('https://www.whatismybrowser.com/detect/what-operating-system-do-i-have')  # test fingerprint

time.sleep(30)

driver.quit()  # Execute on the End!
```

**Reply by ttlns** [reply, marked as answer]
- changing the Platform and Useragent isn't enough for emulatiom
- have a look at https://github.com/kaliiiiiiiiii/Selenium-Profiles/blob/c97ef5183bd0679a8ad3cac289338c3227913054/src/selenium_profiles/profiles/default.json#L35

 doesn't seem to me like a bug

**Comment by ttlns** [comment]
Closing as it seems to be resolved
